src/sdk/pynni/nni/msg_dispatcher.py

```diff
@@ -114,6 +114,7 @@ def handle_import_data(self, data):
         data: a list of dictionaries, each of which has at least two keys, 'parameter' and 'value'
         """
         for entry in data:
+            entry['value'] = entry['value']  if type(entry['value']) is str else json_tricks.dumps(entry['value'])
```

Comment:
better to use `isinstance` instead of `type() is`

Reply:
In this case `type() is` may better, this because we really want to know is whether the type of entry['value'] is `<class 'str'>`. If the type of entry['value'] is a subclass of `str`, `isinstance(entry['value'], str)` will return `true` and  `type(entry['value']) is str` will return `false`. Although `json_tricks.dumps()` will treat subclass of `str` as `str`, I think it is necessary to distinguish subclasses of `str` from `str` at the beginning.

Reply:
got it